plasma_framework/contracts/src/exits/payment/routers/PaymentStandardExitRouter.sol

```diff
@@ -103,10 +103,15 @@ contract PaymentStandardExitRouter is
 
     /**
      * @notice Getter retrieves standard exit data of the PaymentExitGame
-     * @param exitId Exit ID of the standard exit
+     * @param exitIds Exit IDs of the standard exits
      */
-    function standardExits(uint160 exitId) public view returns (PaymentExitDataModel.StandardExit memory) {
-        return standardExitMap.exits[exitId];
+    function standardExits(uint160[] calldata exitIds) external view returns (PaymentExitDataModel.StandardExit[] memory) {
```

Comment:
Unbound loops are not recommended in smart contracts. Let's add another parameter and require the caller to provide how many items he wants to have returned.

Reply:
This function is passing in the exact keys to retrieve data, the size is exactly the size of the array. I don't really see more benefit by passing in the size of array as another parameter here though.

Reply:
If this is just a getter that a clients uses it's fine. If at some point we decided to use this from another contract then the call could run out of gas.

Reply:
> If at some point we decided to use this from another contract then the call could run out of gas.

ehh yes. But my point is that adding another var does not add benefit. The client should just pass in less exitIds in that case.

Reply:
also I presume that because we tagged the function as `external` you can't really use it from another contract, right?

Reply:
oh no lol, tag it `external` make it accessible from outside of that contract. (So not accessible within the contract, weird I know lol)

`public` make it accessible too everything. Just `external` would be slightly gas efficient compare to public

Reply:
It's fine for client, the problem would arise if the call is from another contract. I am ok with @dev comment saying that you need to to be careful if you call this function from another contract,